README.md

```diff
@@ -18,5 +18,9 @@ Gregory will be creating a program to set alarms for times when Benji usually pa
 # LICENSE
 The license I will be using with this project is GNU General Public License v3.0.
 
+
+I would like to add a module going more in depth to the day-to-day routine we get to see with this yorkie.
+=======
```

Comment:
Sounds like a good idea. I could start a day in the life blog and then just document highlights or interesting behaviors.